packages/mds-db/processors.ts

```diff
@@ -0,0 +1,112 @@
+import { StateEntry, TripEntry, MetricsTableRow, Recorded, VEHICLE_EVENT } from '@mds-core/mds-types'
+import schema, { TABLE_NAME } from './schema'
+import { vals_sql, cols_sql, vals_list, logSql } from './sql-utils'
+import { getWriteableClient, makeReadOnlyQuery } from './client'
+
+export async function getStates(
```

Comment:
Based on a convo with @avatarneil I'm told we prefer arg objects rather than ordered args for db query methods.